cpp/modules/deck.gl/core/src/lib/earcut.hpp

```diff
@@ -0,0 +1,823 @@
+// ISC License
```

Comment:
Perhaps we should try to incorporate this in our dependency repo somehow, mainly in order to be able to do version control in a more convenient way.

This should be good for now though

Reply:
Talking with Ib, it was suggested to just copy/paste into deck.gl along with the copyright message at the top of course. Definitely makes sense to later incorporate this into the deps repo.

Reply:
Yes this is fine. You can check the repo but I strongly doubt that there is much activity in this file, it is not like a dependency that we need to keep fresh. We could probably just treat it as a forked file.